core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -245,6 +245,80 @@ class Constants {
    */
   readonly keys_flags_extend = 1;
 
+  /**
+   * Constant for no modifiers
+   */
+  readonly keys_mod_none = 0;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_altL = 1 << 2;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_altR = 1 << 3;
+  /**
+   * bitmask for either alt
+   */
+  readonly keys_mod_alt  = this.keys_mod_altL | this.keys_mod_altR;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_caps = 1 << 7;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_cmd =  1 << 15;
```

Comment:
```suggestion
   * bitmask for Command (Apple) / Windows (Microsoft) modifier key
   */
  readonly keys_mod_cmd =  1 << 15;
```

Do we need a left/right version of this? (Apple have L/R command keys). Note that we do not propose allowing LDML keyboards to use this key, so perhaps we should be excluding it?